webapp/portlet/src/main/webapp/vue-apps/profile-settings/components/drawers/ProfilePropertyLabels.vue

```diff
@@ -71,7 +71,7 @@ export default {
 
   methods: {
     addNewLabel() {
-      this.propertylabels.push( {language: '', label: '', objectId: this.id, objectType: this.labelsObjectType});
+      this.propertylabels.push( {language: 'en', label: '', objectId: this.id, objectType: this.labelsObjectType});
```

Comment:
Wouldn't it be better to select the default language of the user with the property **eXo.env.portal.language** ?

Reply:
done !